code/modules/mob/living/basic/guardian/guardian_verbs.dm

```diff
@@ -0,0 +1,185 @@
+/// Pop out into the realm of the living.
+/mob/living/basic/guardian/proc/manifest(forced)
+	if (is_deployed() || isnull(summoner?.loc) || istype(summoner?.loc, /obj/effect) || (!COOLDOWN_FINISHED(src, manifest_cooldown) && !forced) || locked)
+		return FALSE
+	forceMove(summoner.loc)
+	new /obj/effect/temp_visual/guardian/phase(loc)
+	COOLDOWN_START(src, manifest_cooldown, 1 SECONDS)
+	reset_perspective()
+	manifest_effects()
+	return TRUE
+
+/// Go and hide inside your boss.
+/mob/living/basic/guardian/proc/recall(forced)
+	if (!is_deployed() || isnull(summoner) || (!COOLDOWN_FINISHED(src, manifest_cooldown) && !forced) || locked)
+		return FALSE
+	new /obj/effect/temp_visual/guardian/phase/out(loc)
+	recall_effects()
+	forceMove(summoner)
+	COOLDOWN_START(src, manifest_cooldown, 1 SECONDS)
+	return TRUE
+
+/// Do something when we appear.
+/mob/living/basic/guardian/proc/manifest_effects()
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_GUARDIAN_MANIFESTED)
+
+/// Do something when we vanish.
+/mob/living/basic/guardian/proc/recall_effects()
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_GUARDIAN_RECALLED)
+
+/// Swap to a different mode... if we have one
+/mob/living/basic/guardian/proc/toggle_modes()
+	to_chat(src, span_bolddanger("You don't have another mode!"))
+
+
+/// Turn an internal light on or off.
+/mob/living/basic/guardian/proc/toggle_light()
+	if (!light_on)
+		to_chat(src, span_notice("You activate your light."))
+		set_light_on(TRUE)
+	else
+		to_chat(src, span_notice("You deactivate your light."))
+		set_light_on(FALSE)
+
+
+/// Prints what type of guardian we are and what we can do.
+/mob/living/basic/guardian/verb/check_type()
+	set name = "Check Guardian Type"
+	set category = "Guardian"
+	set desc = "Check what type you are."
+	to_chat(src, playstyle_string)
+
+
+/// Speak with our boss at a distance
+/mob/living/basic/guardian/proc/communicate()
+	if (isnull(summoner))
+		return
+	var/sender_key = key
+	var/input = tgui_input_text(src, "Enter a message to tell your summoner", "Guardian")
+	if (sender_key != key || !input) //guardian got reset, or did not enter anything
+		return
+
+	var/preliminary_message = span_boldholoparasite("[input]") //apply basic color/bolding
+	var/my_message = "<font color=\"[guardian_colour]\"><b><i>[src]:</i></b></font> [preliminary_message]" //add source, color source with the guardian's color
+
+	to_chat(summoner, "<span class='say'>[my_message]</span>")
```

Comment:
we should really add a `span_say()` macro we use this old system like 10 times in this file alone